src/parser/mod.rs

```diff
@@ -6083,6 +6083,24 @@ impl<'a> Parser<'a> {
                 None
             };
             Ok(Some(ColumnOption::Identity(property)))
+        } else if dialect_of!(self is SQLiteDialect)
+            && self.parse_keywords(&[Keyword::ON, Keyword::CONFLICT])
+        {
+            // Support ON CONFLICT for SQLite
+            let on_confl = self.parse_one_of_keywords(&[
+                Keyword::ROLLBACK,
+                Keyword::ABORT,
+                Keyword::FAIL,
+                Keyword::IGNORE,
+                Keyword::REPLACE,
+            ]);
+            match on_confl {
```

Comment:
Would it be better to rename `on_confl` to the full form `on_conflict`?

Reply:
Completely agree, I modified the code accordingly